Makefile

```diff
@@ -83,6 +97,20 @@ all ::
 	@echo "HTML built: `ls -lAFgh $(BUILD_DIR)/manual/build/html/index.html`"
 	@echo "PDF built: `ls -lAFgh $(BUILD_DIR)/manual/build/latex/nexus.pdf`"
 
+$(BASE_CLASS_DIR)/%.nxdl.xml : $(BASE_CLASS_DIR)/$(NYAML_SUBDIR)/%.yaml
```

Comment:
This adds targets to the makefile which are not visible when doing `make help`.

Edit: ok I guess you do not need to see these targets because the `nxdl` target gathers them all.

Reply:
Yes they are basically just executing to build the particular nxdl file but the actual command in this case is `nxdl`